server/container_status.go

```diff
@@ -69,3 +78,33 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	logrus.Debugf("ContainerStatusResponse: %+v", resp)
 	return resp, nil
 }
+
+func (s *Server) getMounts(id string) ([]*pb.Mount, error) {
+	config, err := s.store.GetFromContainerDirectory(id, "config.json")
+	if err != nil {
+		return nil, err
+	}
+	var m rspec.Spec
+	if err = json.Unmarshal(config, &m); err != nil {
+		return nil, err
+	}
+	mounts := []*pb.Mount{}
+	for _, b := range m.Mounts {
+		if b.Type != "bind" {
```

Comment:
This is not valid, because bind mounts are not defined by their `type` in the OCI spec. Rather what we should be checking is whether the string `"bind"` is in the mount options. And we also should be checking for `"rbind"` as well.